src/sentry/incidents/subscription_processor.py

```diff
@@ -514,6 +514,9 @@ def process_update(self, subscription_update: QuerySubscriptionUpdate) -> None:
         self.has_anomaly_detection = features.has(
             "organizations:anomaly-detection-alerts", self.subscription.project.organization
         )
+        fake_anomalies = features.has(
```

Comment:
nit: `has_fake_anomalies` - i just got a bit confused later when i was reading code thinking that these were a list of fake anomalies rather than a boolean flag.